api/trip/serializers.py

```diff
@@ -0,0 +1,27 @@
+from .models import Trip, Activity
+from rest_framework import serializers
+
+
+class TripSerializer(serializers.HyperlinkedModelSerializer):
+    activities = serializers.HyperlinkedRelatedField(
+        view_name='activity_detail',
+        many=True,
+        read_only=True
+    )
```

Comment:
It'll be easier for the frontend if this is a [nested relationship](https://www.django-rest-framework.org/api-guide/relations/#nested-relationships)